recipes/nasm/all/conanfile.py

```diff
@@ -103,15 +95,18 @@ def package(self):
         copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
         if is_msvc(self):
             copy(self, pattern="*.exe", src=self.source_folder, dst=os.path.join(self.package_folder, "bin"), keep_path=False)
+            with chdir(self, os.path.join(self.package_folder, "bin")):
+                shutil.copy2("nasm.exe", "nasmw.exe")
+                shutil.copy2("ndisasm.exe", "ndisasmw.exe")
```

Comment:
Maybe https://docs.conan.io/en/latest/reference/conanfile/tools/files/basic.html?highlight=get#conan-tools-files-rename

Or the tools copy helper... since it's more robust

Reply:
Yes well I don't understand how I'd do this with conan.tools.files.copy(),
I'd have to copy to a temporary folder and then rename into the right place, and then delete the temp folder?
Seems crazy?

I think we should:
- leave it there, with a FIXME or something
- add a conan.tools.files.copy_file() to 1.54
- add a linter check for shutil.copy() and shutil.copy2()

Reply:
That does sound crazy 🤪 lol sorry I didn't realize it was not available 😅